benefits/tech_scheme.md

```diff
@@ -0,0 +1,19 @@
+# Tech-scheme
+
+We are signed up to Techscheme, part of Cyclescheme, which you can upon successful completion of your probation.
```

Comment:
The first sentence in our benefits usually explains why it's a benefit. Might be nice to say something like:

"We signed up to Techscheme to make it easier and more affordable for our team to purchase personal electronics."

I'd remove the detail about its use after probation to somewhere else in the document. Doesn't feel like the most important thing to say first.

We should also put a note somewhere that for all work related computer goods, they should be purchased and expensed instead.

Reply:
Makes sense. Will update to:

'We've signed up to Techscheme to make it easier and more affordable for our team to buy the latest tech gadgets.'